README.md

```diff
@@ -54,15 +54,15 @@ Please see the [Demo Docker Repository](https://hub.docker.com/r/rapidsai/rapids
 
 cuDF can be installed with conda ([miniconda](https://conda.io/miniconda.html), or the full [Anaconda distribution](https://www.anaconda.com/download)) from the `rapidsai` channel:
 
-For `cudf version == 0.13` :
+For `cudf version == 0.18` :
```

Comment:
Can we add these to the release script version updater?

Reply:
Done. Should we just remove this change now?

Reply:
Nah, lets leave it.